### DISCUSSION TITLE
Frigate+ Prepaid 3/6 month & 1 year options?

### DISCUSSION THREAD

**Discussion by Joshndroid** [blakeblackshear/frigate#3548] [Ideas]
I have been using Frigate for a fair while and really happy with the results.
I finally grabbed a coral and its even better, kudos on the awesome program.

I was digging around here and see that you have the frigate+ option which provides some support to the program. I'll be honest in saying that I would like to throw some support, however I am becoming tired of the SaaS monthly subscription model widely used.

What I am wondering is if you will be looking at adding prepaid options for 3/6 months or yearly?

**Comment by blakeblackshear** [comment]
I am definitely looking at those options.

**Reply by Joshndroid** [reply]
Thanks I'll keep an eye out :)

**Comment by lordratner** [comment]
I'll pile on, if Frigate becomes another monthly charge for a service (Netflix, Youtube Premium, Spotify, Hulu, Google Drive, Nest, etc), it'll be time once again to move to a different option. I know, there's a free tier, but inevitably the best features get locked behind a subscription paywall, which is exactly what Frigate+ is doing. 

I'm happy to pay for software. Set a price, and if the price is right, boom. But subscriptions, especially for something that is primarily desirable because it is **self hosted,** are awful. And a "annual pre-paid" option is still a subscription. 

An option to pay one fee for all the current features would be much better. Then if down the road you add more features to a new version, you can charge for the upgrade, giving us the option to stick with what we got or pay for the new toys.

But with a subscription I am now stuck migrating to a new program if you decide in a year to raise the price to $100/month, create more tiers, or change the service in a way I can't avoid by not upgrading. 

I would happily pay a (large) one time fee for a "lifetime subscription" if you'd like to keep the monthly subscription option for those who aren't sure or have limited resources. But a monthly subscription is completely antithetical to the self-hosting philosophy, and it would be a shame to see frigate move in that direction. 

Thanks for an awesome program, Blake, I'm looking forward to seeing the improvements take shape.

Seth
